client/my-sites/stats/summary/index.jsx

```diff
@@ -378,13 +388,19 @@ class StatsSummary extends Component {
 							} }
 							rightSection={
 								<div className="stats-module__header-nav-button">
-									<DownloadCsv
-										skipQuery={ statType === 'utm' }
-										statType={ statType }
-										query={ moduleQuery }
-										path={ path }
-										period={ this.props.period }
-									/>
+									{ shouldGateStatsCsvDownload ? (
```

Comment:
Note: This is enabling stats download gating on UTM and Video summary pages which allowed downloading in production - I need confirmation if this is expected to be gated or those two stats should be allowed?

Reply:
<img width="1348" alt="image" src="https://github.com/user-attachments/assets/55647acc-5787-4685-810c-0d56cd276c4e" />

Reply:
The upsell button on @kangzj's screenshot is for an upgrade to Personal, which might be a bit weird in that case.